examples/garrawarla/configs/packages.yaml

```diff
@@ -0,0 +1,193 @@
+packages:
+  all:
+    target: [broadwell]
+    compiler: [gcc@8.3.0, gcc@4.8.5]
```

Comment:
ensure the list of compilers is consistent with `compilers.yaml`, e.g. right now it's `8.3.0` in packages, and `10.1.0` in compilers